static/src/js/routes/Admin/Admin.jsx

```diff
@@ -59,16 +60,24 @@ export class Admin extends Component {
                   |
                   {' '}
                   <PortalLinkContainer
-                    to="/admin/retrievalsMetrics"
+                    to="/admin/retrievals-metrics"
```

Comment:
Is this url a hyphen or an underscore? I think the serverless.yml had is set to an underscore. Hyphen is more typical, I think.

Reply:
We seem to be using hyphens in React, and underscores in serverless. So think change is bringing things in line with the rest of the app

Reply:
That checks out

Reply:
I mean, it's hyphen now. I think if they aren't they should be.